tests/functional/Sheet/Safe/ExcelExtractorTest.php

```diff
@@ -112,9 +114,16 @@ public function testExtractEmptyFile(): void
 
         $extractor = new Extractor($this->reader, 'Sheet1', 0);
 
-        $this->assertPipelineExtractsLike(
+        $this->assertExtractorExtractsLike(
             [],
             $extractor
         );
     }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)